lib/cartopy/crs.py

```diff
@@ -1140,6 +1140,71 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertEqualArea(Projection):
+    """
+    A Lambert azimuthal equal-area projection.
+
+    """
+
+    def __init__(self, central_longitude=0.0, central_latitude=0.0,
+                 false_easting=0.0, false_northing=0.0,
+                 globe=None):
+        """
+        Kwargs:
+
+            * central_longitude - The central longitude. Defaults to 0.
+            * central_latitude - The central latitude. Defaults to 0.
+            * false_easting - X offset from planar origin in metres.
+                              Defaults to 0.
+            * false_northing - Y offset from planar origin in metres.
+                               Defaults to 0.
+            * globe - A :class:`cartopy.crs.Globe`.
+                      If omitted, a default globe is created.
+
+        """
+        proj4_params = [('proj', 'laea'),
+                        ('lon_0', central_longitude),
+                        ('lat_0', central_latitude),
+                        ('x_0', false_easting),
+                        ('y_0', false_northing)]
+
+        super(LambertEqualArea, self).__init__(proj4_params, globe=globe)
+
+        a = np.float(self.globe.semimajor_axis or 6378137.0)
+        b = np.float(self.globe.semiminor_axis or 6356752.3142)
+        x_axis_offset = 1.3e7 / 6378137.
+        y_axis_offset = 1.3e7 / 6356752.3142
+        self._x_limits = (-a * x_axis_offset + false_easting,
+                            a * x_axis_offset + false_easting)
```

Comment:
The theoretical X axis limits are ±a * 2 + false_easting. But as with other projections it's best to reduce those limits a little to avoid problems at the boundary, e.g. `(-a * 1.99999 + false_easting, a * 1.99999 + false_easting)`.